docs/how-to-guides/propagating-updates.md

```diff
@@ -0,0 +1,64 @@
+
+Generally speaking, this is the preferred approach due to the overhead involved with managing [block deprecations](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-deprecation/) and the requirement to manually update existing blocks otherwise. Keep in mind that the biggest downside with dynamic blocks comes from needing to manage markup in two places. Depending on your needs and comfortability, either approach can work. 
+
+**Patterns**
+
+For content that you want updated later on, do not use patterns and instead rely on reusable blocks or template parts. Patterns cannot be updated after you insert one into your site. For context, you can think of Patterns as more like sample/example/starter content. While they allow you to quickly populate a post or template, as soon as they are inserted, they become disconnected from the source of truth you’re inserting from. 
+
+If needed, one potential workaround for patterns with custom styles is to add a class name to the wrapping block for a pattern. For example, the following adds a themeslug-special class to a Group block:
+
+```
+<!-- wp:group {"className":"themeslug-special"} -->
+<div class="wp-block-group themeslug-special">
+	<!-- Nested pattern blocks -->
+</div>
+<!-- /wp:group -->
+```
+
+It is not fool-proof because users can modify the class via the editor UI.  However, because this setting is under the "Advanced" panel it is likely to stay intact in most instances. This gives theme authors some CSS control for some pattern types, allowing them to update existing uses. However, it does not prevent users from making massive alterations that cannot be updated. 
+
+**Reusable blocks**
+
+As the name suggests, these blocks are inherently synced across your site. Keep in mind that there are currently limitations with relying on reusable blocks to handle certain updates since content, HTML structure, and styles will all stay in sync when updates happen. If you require more nuance than that, this is a key element to factor in and a dynamic block might be a better approach.
+
+**Template Parts and Templates**
+
+When templates or template parts are changed by the user, the updated templates from the theme update don’t show for the user. Only new users of the theme or users who have not yet edited a template are experiencing the updated template. If users haven’t modified the files then the changes you make on the file system will be reflected on the user’s sites – you just need to update the files and they’ll get the changes. However if they have made changes to their templates then the only way you can update their templates is to:
```

Comment:
So this section explains how the block theme works in regard to local changes applied to the templates and template parts. It took me some time to understand that. Maybe adding an introduction that sets the proper context would make that clear from start.

Reply:
I added in a sentence more of context. Let me know if more is needed.